### ISSUE TITLE
Attrs dependency requirement conflicts with Airflow

### ISSUE THREAD

**Issue by kemics** [Toloka/toloka-kit#37]
Hi, thank you for the helpful library

I am trying to use toloka-kit in Airflow environment. However, when I am installing both `apache-airflow` and `toloka-kit` packages it fails with dependency conflict error. Something like this:

`apache-airflow 2.1.2+composer has requirement attrs<21.0,>=20.0, but you have attrs 21.2.0.`

I tried different recent versions of Airflow (both v1 and v2 and `master` branch) and all are failing with a similar error

And it looks like that `toloka-kit` relies on API of new 21 version of `attrs`. Because I ran `toloka-kit` tests in virtual env with `attrs==20.3.0` and 26 tests failed.

I understand that it would be unpleasant to downgrade `attrs` package version requirement here. However, Airflow is a very popular solution and I assume that I am not the only one who manages or will manage Toloka projects via Airflow. Moreover, I doubt that Airflow will update the dependency version soon. Currently this is a blocker for me and maybe to somebody else to migrate from bare API calls to `toloka-kit`. For example, I am not able to create a [hook](https://airflow.apache.org/docs/apache-airflow/stable/concepts/connections.html#hooks), `PythonOperator` or `PythonSensor` for crowdsourcing purposes via `toloka-kit`

What do you think?

**Comment by Losik** [comment]
The reason for `attrs>=21.2.0` is this feature https://github.com/python-attrs/attrs/pull/710 that we rely on for correct signatures iand correct stub generation.This feature should have appeared in `20.4.0` but the only unyanked release after `20.3.0` is `21.2.0`. 

I guess we can downgrade to `attrs>=20.3.0`. If we use later versions of attrs in our environment during stub-generation process. The only thing this would affect is `inspect.signature(toloka_client.<method_name>)` which would produce correct but less detailed signatures (having `Any` on positions that would otherwise be more specific). IDE hints will stay detailed, though.

long story short, we'll downgrade to `attrs>=20.3.0` in the next release

**Comment by kemics** [comment]
@Losik thank you again for solving this so fast. Can you please push tag (looks like 0.1.15)?

**Comment by Losik** [comment]
Yes, I must do one more commit with changelog updates and version increase, merge it with main branch and then create a new release. This commit have to sync with GitHub from our internal repo and this action is triggered once an hour. Unfortunately, this functionality is maintained by another team and I do not have a way to trigger this synchronization manually at the moment =( 

Long story short, I expect new PyPI release to be ready in about an hour

**Comment by Losik** [comment]
@kemics, toloka-kit==0.1.15 is in PyPI :)

**Comment by kemics** [comment]
Thanks a lot, this change really helps to use this package in my production 🎉

**Comment by Pocoder** [comment]
@kemics, if you continue to use toloka-kit in airflow, you can be early adopter and try out our new [library for airflow](https://github.com/Toloka/toloka-airflow). We will be very grateful for your feedback =)
